softgroup/services/authorization/authorization-api/src/main/java/com/softgroup/authorization/api/message/SmsConfirmRequest.java

```diff
@@ -0,0 +1,29 @@
+package com.softgroup.authorization.api.message;
+
+import com.softgroup.common.protocol.RequestData;
+
+/**
+ * Created by nikmlk on 24.02.17.
+ */
+public class SmsConfirmRequest implements RequestData {
+    private static final long serialVersionUID = 16282141842368961L;
+
+    private String auth_code;
```

Comment:
@melnyknikolay Maybe when you copy something - it would be useful to read what you copy.